nowradio/dbManager.py

```diff
@@ -55,7 +55,7 @@ def __init__(self):
         FROM station_info
         WHERE our_genre = %(genre_name)s AND is_up AND NOT blacklisted
         GROUP BY name
-        ORDER BY max(active_listeners + faves) DESC
+        ORDER BY max((%(active)s * active_listeners) + (%(fav)s * faves) - (%(timeout)s * timeout_count)) DESC
```

Comment:
let's make the constant names more clear, e.g. `active_listeners_weight`, `faves_weight`, `timeout_count_weight`